src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const splittedArr = sourceString.split(';');
```

Comment:
Consider using a more descriptive variable name instead of 'splittedArr'. For example, 'styleDeclarations' would more accurately describe the contents of the array.